airbyte-integrations/connectors/source-intercom/manifest.yaml

```diff
@@ -243,10 +243,14 @@ definitions:
         type: CustomRetriever
         class_name: "source_declarative_manifest.components.IntercomScrollRetriever"
         requester:
+          type: CustomRequester
+          class_name: source_declarative_manifest.components.NoCacheRequester
           $parameters:
             name: "companies"
-          $ref: "#/definitions/base_requester"
+          url_base: https://api.intercom.io/
```

Comment:
why the change to `url_base` and `authenticator`?

Reply:
Because we're now using a custom requester expressed by `class_name`, we seemed to lose the configuration that was coming from the `$ref` import-like. Not super confident on this but I got an error without redefining these values.